### PR TITLE
Add dm_control recipe

### PR DESCRIPTION
**Pull request by traversaro**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by traversaro** [comment]
I did not noticed that labmaze dependency, that is not trivial to package (several C++ deps vendored via bazel) and not packaged on conda-forge . I am closing this PR, we can resume it one we have labmaze .

**Comment by h-vetinari** [comment]
For 
```
self = <dm_control._render.executor.render_executor.PassthroughRenderExecutor object at 0x7f6bcd7b4af0>
func = <class 'mujoco._render.MjrContext'>
args = (<mujoco._structs.MjModel object at 0x7f6bcfa735f0>, <mjtFontScale.mjFONTSCALE_150: 150>)
kwargs = {}

    def call(self, func, *args, **kwargs):
      self._check_locked()
>     return func(*args, **kwargs)
E     mujoco.FatalError: gladLoadGL error
```
are we expecting these to pass in CI?

E.g. mujoco feedstock skips the render tests, but here it looks we're also in a rendering context.

File: recipes/dm_control/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    - scipy
+    - tqdm
+    - pillow
+
+test:
+  imports:
+    - dm_control
+  commands:
+    - pip check
+    - pytest --pyargs dm_control -v -k "not (viewer or renderer or render)"
```
Comment (h-vetinari) [inline recipes/dm_control/meta.yaml:50]: pytest does very simple string matching on the serialization of the test here, e.g.
```
locomotion/tasks/corridors_test.py::CorridorsTest::test_termination_and_discount
```

"renderer" and "render" are redundant, and don't capture the entirety of tests that call into render code (i.e. the above example of a failing test contains neither "viewer" nor "render" in its serialization).

Unless dm_control has some pytest tags that can be used to skip various classes of issues, you're down to building a long chain of test names, e.g. [like so](https://github.com/conda-forge/scipy-feedstock/blob/main/recipe/meta.yaml#L72-L101).

Comment (h-vetinari) [reply recipes/dm_control/meta.yaml]: such long lines are not necessarily more readable but 🤷 

Glad you found a way to fix it. :)

**Comment by traversaro** [comment]
So, after excluding the strings of tests that are failing because:
* A GL context cannot be created
* On Windows some tests assets cannot be founded

We still have ~1900 tests out of ~2700 test passing, that means that at least a good part of the package is working correctly. At this point I think we can consider the recipe ready for review. @conda-forge/help-python

**Comment by traversaro** [comment]
Thanks @carterbox !

**Pull request merged**